addons/crm/models/crm_lead.py

```diff
@@ -2083,11 +2085,27 @@ def _pls_get_naive_bayes_probabilities(self, batch_mode=False):
         To avoid this, we add 0.1 in each frequency. With few data, the computation is than not really realistic.
         The more we have records to analyse, the more the estimation will be precise.
         :return: probability in percent (and integer rounded) that the lead will be won at the current stage.
+
+        If is_tooltip is True, method instead recomputes the probability of self, that should be a singleton, and instead
+        returns a dict containing team name, probability, and a list of all (score, field, value) triplets for all value
+        of PLS fields that impact the computation of the probability. Score is a simple value that indicates whether the
+        impact is positive (>.5) or negative (<.5).
         """
         lead_probabilities = {}
         if not self:
             return lead_probabilities
 
+        # Initialize tooltip data. A returned 0.00 probability means computation was not possible.
+        tooltip_data = {}
+        if is_tooltip:
+            if len(self) != 1:
+                raise ValidationError(_("PLS Tooltip is only supported for a single lead at a time"))
```

Comment:
Do we really need this type of error message, could just be an `ensure_one` instead imo